MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+        <StackLayout VerticalOptions="CenterAndExpand">
+          <Label Text="&#xe725;" 
+                 FontSize="40"
+                 FontFamily="{OnPlatform iOS=ListViewFontIcons, MacCatalyst=ListViewFontIcons, Android=ListViewFontIcons.ttf#, UWP=ListViewFontIcons.ttf#ListViewFontIcons}"
+                 HorizontalTextAlignment="Center"/>
+          <Label Text="{Binding .,StringFormat='{0} is not found'}" 
+                 FontSize="16" 
+                 FontFamily="Roboto-Regular" 
+                 HorizontalTextAlignment="Center"/>
+        </StackLayout>
+      </DataTemplate>
+      <local:EmptyViewDataTemplateSelector x:Key="DataTemplateSelector" 
+                                           BasicTemplate="{StaticResource BasicTemplate}" 
+                                           AdvancedTemplate="{StaticResource AdvancedTemplate}"/>
+    </ResourceDictionary>
+  </ContentPage.Resources> 
+
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="30"/>
+      <RowDefinition Height="*" />
+    </Grid.RowDefinitions>
+    <SearchBar x:Name="filterText" 
+               FontSize="16"                              
+               Placeholder="Filter Inventory" TextChanged="SearchBar_TextChanged"/>                                                   
+    <syncfusion:SfListView Grid.Row="1" x:Name="listView"
+                           ItemsSource="{Binding Items}"
+                           ItemSize="56"
+                           EmptyView="{Binding Source={x:Reference filterText},Path=Text}"
+                           EmptyViewTemplate="{StaticResource DataTemplateSelector}">                         
+    </syncfusion:SfListView>
+  </Grid>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EmptyView = filterText.Text;
+
+listView.EmptyViewTemplate = new EmptyViewDataTemplateSelector() { BasicTemplate = this.Resources["BasicTemplate"] as DataTemplate, AdvancedTemplate = this.Resources["AdvancedTemplate"] as DataTemplate };
+{% endhighlight %}
+{% endtabs %}
+
+The `SfListView.EmptyView` property is set to the SearchBar.Text property, and the `SfListView.EmptyViewTemplate` property is set to a EmptyViewDataTemplateSelector object.
+
+{% tabs %}
+{% highlight c# tabtitle="EmptyViewDataTemplateSelector.cs" %}
+public class EmptyViewDataTemplateSelector : Microsoft.Maui.Controls.DataTemplateSelector
+{
+  public DataTemplate BasicTemplate { get; set; }
+  public DataTemplate AdvancedTemplate { get; set; }
+
+  public EmptyViewDataTemplateSelector()
+  {
+    
+  }
+
+  protected override DataTemplate OnSelectTemplate(object item, BindableObject container)
+  {
+    if(item.ToString().Count() > 10)
+        return AdvancedTemplate;
+    else
+        return BasicTemplate;
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Template for `SfListView.EmptyView` is set to AdvancedTemplate when SearchBar.Text.Count() is greater than 10. Otherwise, set to BasicTemplate.

```

Comment:
Template for `SfListView.EmptyView` is set to AdvancedTemplate when SearchBar.Text.Count() is greater than 10. Otherwise, set it to BasicTemplate.